ansible/playbooks/server_and_nuc_deploy.yml

```diff
@@ -48,6 +48,46 @@
       include_role:
         name: installation/playbook_setup
       when: not skip_molecule_task|bool
+
+    - name: Set scene and initial_z for production (Bimanual)
+      set_fact:
+        scene_file: "$(find sr_description_common)/scenes/customer_stand_bimanual.scene"
```

Comment:
wouldn't you just have the `customer_stand_bimanual` as the file name and then input that in both world and scene below